tests/parser/features/test_logging.py

```diff
@@ -448,3 +448,32 @@ def ioo(inp: bytes <= 100):
     c.ioo(b"moo4")
     assert s.head_state.receipts[-1].logs[0].data == b'moo4'
     print("Passed raw log tests")
+
+
```

Comment:
Can you add a test with a list as function input?  Can you also add a test with a list of a different type?